sdlhmi

```diff
@@ -0,0 +1,4 @@
+#! /usr/bin/env bash
+
+cd /usr/share/sdlhmi
```

Comment:
@LuxoftAKutsan maybe move this file to DEB folder as well?

Reply:
deb folder has it's own structure, we can't put additional scripts there.